implement 'cl write <target> <string>'

percyliang opened the pull request:
Fixes #199 
@kashizui please review

percyliang committed:
implement 'cl write <target> <string>'

kashizui commented on codalab/machines/local_machine.py:
```diff
@@ -59,10 +60,13 @@ def start_bundle(self, bundle, bundle_store, parent_dict, username):
+        if action_string == BundleAction.KILL:
+            self.process.kill()
+        else:
+            print '=== Unhandled command: %s' % action_string
```
nit: maybe `>>stderr`?

kashizui commented on codalab/lib/bundle_action.py:
```diff
@@ -0,0 +1,17 @@
+    KILL = 'kill'
+    WRITE = 'write'
+
+    """
```
nit: docstring should be on first line

kashizui commented on codalab/lib/bundle_action.py:
```diff
@@ -0,0 +1,17 @@
+
+    @staticmethod
+    def write(subpath, string):
+        return ' '.join([BundleAction.WRITE, subpath, string])
```
Will there be any issues with subpaths with spaces?

percyliang replied:
Good point - will enforce that subpath isn't arbitrary

kashizui commented:
LGTM modulo comments

percyliang committed:
- Restrict 'subpath' to not include arbitrary characters
- Use tab to separate arguments in a bundle action
- Hold on to action if bundle not running yet
- Changed --request_* to --request-* for dispatch commands
